app/lib/report_helper.rb

```diff
@@ -834,20 +834,23 @@ def compute_genera_aggregate_scores!(rows, tax_2d)
       species_score = species_info['NT']['aggregatescore']
       genus_score = genus_info['NT']['aggregatescore']
       unless genus_score && genus_score > species_score
+        # genus aggregate score is the max of the specifies scores
         genus_info['NT']['aggregatescore'] = species_score.to_f
         genus_info['NR']['aggregatescore'] = species_score.to_f
       end
     end
   end
 
   def compute_species_aggregate_scores!(rows, tax_2d)
+    tsm = TaxonScoringModel.find_by(name: TaxonScoringModel::DEFAULT_MODEL_NAME)
     rows.each do |species_info|
       species_info['NT']['maxzscore'] = [species_info['NT']['zscore'], species_info['NR']['zscore']].max
       species_info['NR']['maxzscore'] = species_info['NT']['maxzscore']
       next unless species_info['tax_level'] == TaxonCount::TAX_LEVEL_SPECIES
```

Comment:
Can you invert this to an if that encompasses the code below?
ifs always preferred to unless and continues.